src/main/java/org/opentripplanner/routing/vehicle_parking/VehicleParking.java

```diff
@@ -0,0 +1,145 @@
+package org.opentripplanner.routing.vehicle_parking;
+
+import java.util.List;
+import java.util.Locale;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.opentripplanner.common.RepeatingTimePeriod;
+import org.opentripplanner.model.FeedScopedId;
+import org.opentripplanner.routing.vertextype.StreetVertex;
+import org.opentripplanner.util.I18NString;
+
+/**
+ * Vehicle parking locations, which may allow bicycle and/or car parking.
+ *
+ * All fields are immutable except for the availability, capacity which may be updated by updaters.
+ * If any other properties change a new VehicleParking instance should be created.
+ */
+@Builder
+@Getter
+@EqualsAndHashCode
+public class VehicleParking {
+
+  /**
+   * The id of the vehicle parking, prefixed by the source(=feedId) so that it is unique.
+   */
+  private final FeedScopedId id;
+
+  private final I18NString name;
+
+  /**
+   * Note: x = Longitude, y = Latitude
+   */
+  private final double x, y;
+
+  /**
+   * Whether this station has space available information updated in real-time.
+   */
+  private final boolean realTimeData = true;
+
+  private final String detailsUrl;
+
+  private final String imageUrl;
+
+  /**
+   * Source specific type of the vehicle parking.
+   */
+  private final String type;
```

Comment:
What is the purpose of this? Isn't this covered by the tags?

Reply:
It is, so it's been removed.